CHANGELOG

```diff
@@ -1,3 +1,12 @@
+* Thu Mar 08 2018 Nick Miller <nick.miller@onyxpoint.com> - 6.0.4-0
+- Manage `/etc/ntp/step-tickers`
```

Comment:
I think the changes may warrant a minor version bump instead of a patch bump.

Reply:
:+1: